certora/specs/EnumerableMap.spec

```diff
@@ -66,13 +66,13 @@ invariant atUniqueness(uint256 index1, uint256 index2)
     }
 
 /*
-┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
-│ Invariant: index <> value relationship is consistent                                                                │
-│                                                                                                                     │
-│ Note the two consistencyXxx invariants, put together, prove that at_ and _positionOf are inverse of one another.    │
-│ This proves that we have a bijection between indices (the enumerability part) and keys (the entries that are set    │
-│ and removed from the EnumerableMap).                                                                                │
-└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+┌───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
```

Comment:
these lines are designed to be 119 char long so that they show correctly in a 120 column terminal or in a IDE that sets line wrapping at 120 chars. We should avoid making them longer.

120 is also what we have set in .prettierrc

Reply:
lmao I got this so wrong 😂